cloud-stream-batch-consume/src/main/resources/application.yaml

```diff
@@ -0,0 +1,28 @@
+spring:
+  cloud:
+    function:
+      definition: batchConsume
+    stream:
+      bindings:
+        batchConsume-in-0:
+          destination: 'batch/consume/topic'
+          group: batch
+          content-type: 'application/octet-stream'
+          consumer:
+            batch-mode: true
+            useNativeDecoding: true
```

Comment:
I'm honestly not sure what this does. is it required here? It also says if this is specified it doesn't use `content-type` which is specified above. 

Suggestion: just switch to using a message with a `String` payload to keep it simple. Plus it will work with teh batch publisher that uses messages with `String` payloads.

Reply:
You are right - I did these to get it working because the data was coming in as byte[] by default from Try-Me.

Now, tried with just useNativeEncoing:true (no content-type), and changing the consumer function to accept List<String> - it just works.

Made the changes!